src/main/java/com/iridium/iridiumteams/configs/Commands.java

```diff
@@ -104,5 +105,6 @@ public Commands(String permissionBase, String team, String commandBase) {
         rewardsCommand = new RewardsCommand<>(Collections.singletonList("rewards"), "View your " + team + " rewards", "%prefix% &7/" + commandBase + " rewards", "", 0);
         experienceCommand = new ExperienceCommand<>(Collections.singletonList("experience"), "View your " + team + " experience", "%prefix% &7/" + commandBase + " experience <give/set/remove> <player> <amount>", "", 0, permissionBase + ".experience.modify");
         shopCommand = new ShopCommand<>(Collections.singletonList("shop"), "Open the Shop", "%prefix% &7/" + commandBase + " shop", "", 0);
+        blockValueCommand = new BlockValueCommand<>(Collections.singletonList("blockvalues"), "View block values","%prefix% &7/" + commandBase + "blockvalues <blocks/spawners> <player>", "", 0);
```

Comment:
```suggestion
        blockValueCommand = new BlockValueCommand<>(Collections.singletonList("blockvalues"), "View block values","%prefix% &7/" + commandBase + "blockvalues (team) <blocks/spawners>", "", 0);
```

Reply:
what would be the point of reversing this? iirc its currently set to accept which type you want first

Reply:
Your correct, im dumb and I cant read code

Reply:
```suggestion
        blockValueCommand = new BlockValueCommand<>(Collections.singletonList("blockvalues"), "View block values","%prefix% &7/" + commandBase + "blockvalues <blocks/spawners> (player)", "", 0);
```
Should be this though, since player is an optional argument

Reply:
i didnt know there was a difference tbh